src/main/java/filemanager/serviceImpl/ConverterFromJsonToXmlServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package filemanager.serviceImpl;
+
+import com.google.inject.Inject;
+import filemanager.model.XmlModel;
+import filemanager.service.ConverterFromJsonToXmlService;
+import filemanager.service.JsonReader;
+import filemanager.service.XmlWriter;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class ConverterFromJsonToXmlServiceImpl implements ConverterFromJsonToXmlService {
+
+    private JsonReader reader;
+
+    private XmlWriter writer;
+
+    @Inject
+    public ConverterFromJsonToXmlServiceImpl(JsonReader reader, XmlWriter writer) {
+        this.reader = reader;
+        this.writer = writer;
+    }
+
+    @Override
+    public void readJsonConvertToXmlAndWrite(Path from, String to) throws IOException {
+        XmlModel xmlModel = reader.readJson(new FileInputStream(String.valueOf(from)));
+        String client = reader.getClientFromJson(new FileInputStream(String.valueOf(from)));
+        String temporaryPath = to + client;
```

Comment:
You don't need to parse the file second time to get client name. Just use the one from Interaction. And please change XmlModel name to Interaction